src/config/config.ts

```diff
@@ -68,5 +71,14 @@ export interface Config {
   /**
    * Sandbox feature configuration
    */
-  sandboxFeatures?: SandboxFeatures;
+  sandboxFeatures?: SandboxFeatures | TestFeatures;
```

Comment:
We should not need to use TestFeatures here

Reply:
This was an attempt to resolve this issue during the build:
<img width="767" alt="Screenshot 2024-06-14 at 10 20 00" src="https://github.com/entur/enki/assets/85493679/3e71f133-1b9e-4d9b-a346-3eac5fb6619c">
But maybe should be something else

Reply:
I think you have these problems because of the changes in src/ext/__tests__/SandboxFeature.test.tsx

This is why I extended the SandboxFeature component type.

I can have a look at the branch if you want

Reply:
But it was also occurring earlier, e.g. at the time of commit 2823a87145f514e9e28826ed9b7f11d35ea1bc9b, in a way that's how that chain of changes in the test started

Reply:
Here's a commit that isolates test related code in test files https://github.com/entur/enki/commit/af4be048ec8cab58a16b83fc8a6eaa3786694715

It's acceptable to ignore the typescript error on the config provider, since it's not the part that's being tested here.

Reply:
Looks great 👍 Got it cherry-picked